python/tvm/relay/frontend/pytorch.py

```diff
@@ -1322,6 +1332,16 @@ def avg_pool2d(self, inputs, input_types):
         ceil_mode = int(inputs[4])
         count_include_pad = int(inputs[5])
 
+        if isinstance(pool_size, list):
+            for i in range(len(pool_size)):
+                if isinstance(pool_size[i], _expr.Constant):
+                    pool_size[i] = int(_infer_value_simulated(pool_size[i], {}).asnumpy())
+
+        if isinstance(strides, list):
+            for i in range(len(strides)):
+                if isinstance(strides[i], _expr.Constant):
+                    strides[i] = int(_infer_value_simulated(strides[i], {}).asnumpy())
+
```

Comment:
Let's introduce a helper function to remove all the dups.

Reply:
Done. Thanks. Changed to any type of expr as well.